cms/bundles/tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+
+from cms.articles.models import StatisticalArticlePage
+from cms.articles.tests.factories import StatisticalArticlePageFactory
+from cms.bundles.tests.factories import BundleFactory, BundlePageFactory
+from cms.bundles.utils import get_bundleable_page_types, get_pages_in_active_bundles
+
+
+class BundlesUtilsTestCase(TestCase):
+    def test_get_bundleable_page_types(self):
+        self.assertListEqual(get_bundleable_page_types(), [StatisticalArticlePage])
+
+    def test_get_pages_in_active_bundles(self):
+        self.assertListEqual(get_pages_in_active_bundles(), [])
+
+        bundle = BundleFactory()
+        published_bundle = BundleFactory(released=True)
```

Comment:
The test picked this up. Did you mean to set the state to `BundleStatus.PUBLISHED` instead?

Reply:
Just got back to this. Fixed in 3a0f045. The original commit (c03aa29) went in before #157 and I missed in after the rebase